cmd/ef_tests/blockchain/Cargo.toml

```diff
@@ -29,6 +29,8 @@ default = ["c-kzg", "blst"]
 blst = ["ethrex-vm/blst"]
 c-kzg = ["ethrex-blockchain/c-kzg"]
 levm = []
+sp1 = ["zkvm_interface/sp1", "ethrex-prover/sp1"]
+stateless = []
```

Comment:
Is this feature flag needed? Couldn't this be the default when `sp1` is disabled?